plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergPageSink.java

```diff
@@ -439,6 +442,16 @@ private Optional<PartitionData> getPartitionData(List<PartitionColumn> columns,
         return Optional.of(new PartitionData(values));
     }
 
+    private static Block getPartitionBlock(PartitionColumn column, Page page)
```

Comment:
Seems to be the same as another `private Block getPartitionBlock(PartitionColumn column, Page page)` later in this file